content/blog/mirai-2-5-0/index.Rmd

```diff
@@ -0,0 +1,227 @@
+
+- Connect to different sets of daemons (persistent parallel processes) independently
+- Set up parallelism on your laptop, on a local server, and on an HPC cluster - all at the same time
+- Distribute compute to the most suitable resource for the job
+
+Example:
+```{r, eval=FALSE}
+# Local parallelism
+daemons(4, .compute = "local")
+
+# Multi-server deployment
+ssh_cfg <- ssh_config(c("ssh://server1", "ssh://server2", "ssh://server3"))
+daemons(url = host_url(), remote = ssh_cfg, .compute = "server")
+
+# HPC cluster deployment
+hpc_cfg <- cluster_config(options = "#SBATCH --partition=gpu")
+daemons(100, url = host_url(), remote = hpc_cfg, .compute = "cluster")
+
+# Switch between compute profiles seamlessly
+result1 <- mirai(slow_task(), .compute = "local")
+result2 <- mirai(long_task(), .compute = "server")
+result3 <- mirai(heavy_computation(), .compute = "cluster")
+```
+
+For further details on using mirai in HPC environments, please refer to our last [blog post for mirai 2.4.0](https://shikokuchuo.net/posts/27-mirai-240/), where we introduced our cluster launcher `cluster_config()`.
+
+## New in mirai 2.5.0
+
+### OpenTelemetry integration
+
+We're absolutely delighted to unveil: complete observability of mirai requests through OpenTelemetry traces. This is a critical feature that completes the final pillar in mirai's 'built for production' design philosophy.
+
+When tracing is enabled via the otelsdk package, you can monitor the entire lifecycle of your async computations, making it easier to debug and optimize performance in production environments.
+
+This is especially powerful when used in conjunction with other otel-enabled packages (like an upcoming Shiny release), providing end-to-end observability across your entire application stack.
+
+![*Illustrative span structure shown in a Jaeger collector UI*](otel-screenshot.png)
+
+The hierarchical tracing structure allows you to track the flow from initiation through to final evaluation, enabling precise performance analysis and bottleneck identification.
+
+### Reproducible parallel RNG
+
+We're equally delighted to announce reproducible parallel random number generation. Developed in consultation with our tidymodels colleagues and core members of the mlr3 team, this is a great example of the R community pulling together to implement crucial infrastructure.
+
+Technically this feature has been around since the recent 2.4.1 release, but it solves a long-standing challenge in parallel computing in R. We hope that by announcing it here, it's put to good use for reproducible science.
+
+mirai has long used L'Ecuyer-CMRG streams (since prior to v1.0) for statistically-sound parallel RNG. Streams essentially cut into the RNG’s period (a very long sequence of pseudo-random numbers) at intervals that are far apart from each other that they do not in practice overlap. This ensures that statistical results obtained from parallel computations remain correct and valid. 
```

Comment:
I think you also need to describe what was missing.

Reply:
Made this clear in [395301f](https://github.com/tidyverse/tidyverse.org/pull/755/commits/395301f0f9269c589e53aeec3a4c6fbd01eef19e).